src/celeritas/optical/surface/SurfacePhysicsView.hh

```diff
@@ -19,12 +17,11 @@ namespace optical
 {
 //---------------------------------------------------------------------------//
 /*!
- * Optical surface physics data for a track.
+ * Persistent optical surface physics data for a track.
```

Comment:
In the EM loop we have "step view" and "track view" for persistent track-dependent data that's ephemeral versus long-lived (more or less, anyway). ParticleView depends on particle ID, not track slot ID; and ParticleTrackView depends on track slot ID (and has state). 

I think we need to make this class a true "surface physics view" that encapsulates only params-related data for a single surface. State-related data moves to SurfacePhysicsTrackView. We can add a `SurfaceCrossingTrackView` (or whatever; some more "ephemeral" state) for properties that only matter when entering/interacting with a surface.